guides/02_getting-started/04_blog-tutorial.html.md

```diff
@@ -410,64 +433,99 @@ $ padrino g migration AddAccountToPost account_id:integer
 ```
 
 This creates a new migration with the desired field attaching the `account_id` to
-the post.
-
-Let's modify the migration file to assign a user to all existing posts:
+the post. Let's modify this migration to add the accounts FK to posts:
 
 ```ruby
 # db/migrate/003_add_account_to_post.rb
-class AddAccountToPost < ActiveRecord::Migration
-  def self.up
-    change_table :posts do |t|
-      t.integer :account_id
+
+Sequel.migration do
+  class Account < Sequel::Model; end
+  class Post < Sequel::Model
+    many_to_one :account
+  end
+
+  up do
+    alter_table :posts do
+      add_column :account_id, Integer
     end
-    # and assigns a user to all existing posts
-    first_account = Account.first
-    Post.all.each { |p| p.update_attribute(:account, first_account) }
   end
 
-  def self.down
-    change_table :posts do |t|
-      t.remove :account_id
+  down do
+    alter_table :posts do
+      drop_column :account_id
     end
   end
 end
 ```
 
+
 Now, we'll return to the Post Model to setup the `account` association and add a
 few validations.
 
 ```ruby
-# app/models/post.rb
-class Post < ActiveRecord::Base
-  belongs_to :account
-  validates_presence_of :title
-  validates_presence_of :body
+# models/post.rb
+
+class Post < Sequel::Model
+  many_to_one :account
+
+  plugin :validation_helpers
+  def validate
+    super
+    validates_presence [:title, :body]
+  end
 end
 ```
 
-Before running the migration we need to add the `models` folder to the `dependency\_paths`:
+And add the association to the Account model:
+
 
 ```ruby
-# config/boots.rb
-Padrino.before_load do
-  Padrino.dependency_paths << Padrino.root('models/**/*.rb')
+class Account < Sequel::Model
+  one_to_one :post
+  ...
 end
 ```
 
-Without it, the migration cannot find the `Account.first`. Now we are ready
-to run the migration:
+Now we are ready to run the migration: `$ padrino rake sq:migrate`
+
+Let's create another migration to assign the first user to all existing posts:
 
 ```shell
-$ padrino rake db:migrate
-=> Executing Rake db:migrate ...
-   INFO -  Migrating to AddAccountToPost (3)
-  DEBUG -   (0.1ms)  begin transaction
-== 3 AddAccountToPost: migrating ==============================================
--- change_table(:posts)
+$ padrino g migration MigrateExistingPostsToFirstAccount
+       apply  orms/activerecord
+      create  db/migrate/004_migrate_existing_posts_to_first_account.rb
+```
+
+And change the content of the migration:
+
+```ruby
+# db/migrate/004_migrate_existing_posts_to_first_account.rb
+Sequel.migration do
+  class Account < Sequel::Model; end
+  class Post < Sequel::Model
+    many_to_one :account
+  end
```

Comment:
Let's drop the class definitions from this migration too. I can't see them being required.